rpm/sdk-client.spec

```diff
@@ -34,6 +34,8 @@ make %{?_smp_mflags}
 %install
 rm -rf %{buildroot}
 %make_install
+mkdir -p  %{buildroot}/%{_sysconfdir}/systemd/system/sockets.target.wants
+ln -s /usr/lib/systemd/system/sshd.socket %{buildroot}/%{_sysconfdir}/systemd/system/sockets.target.wants/
```

Comment:
I believe the link was originally created with the `%systemd_post sshd.socket` below, works when this and the openssh-server package are installed in the correct order, like in 3.2.0.24:

```
Info[10/06 20:59:23] : Installing: openssh-server       ++++++++++++++++++++++++           [420/577]
...
Info[10/06 20:59:23] : Installing: sdk-client-emul      ++++++++++++++++++++++++           [424/577]
```

In devel the order is opposite, in which case `%systemd_post` silently fails:

```
Info[11/03 10:51:43] : Installing: sdk-client-emul ++++++++++++++++++++++++ [403/567]
...
Info[11/03 10:51:44] : Installing: openssh-server +++++++++++++++++++++++++ [431/567]
```

This should be fixed by adding `Requires(post): openssh-server` I think.

Reply:
> This should be fixed by adding `Requires(post): openssh-server` I think.

I'll try that, thanks!